lib/vanagon/platform/windows.rb

```diff
@@ -44,6 +44,9 @@ def generate_packaging_artifacts(workdir, name, binding)
         ["chocolateyInstall.ps1", "chocolateyUninstall.ps1"].each do |win_file|
           FileUtils.copy(File.join(VANAGON_ROOT, "resources/windows/nuget/#{win_file}"), File.join(workdir, win_file))
         end
+
+        # MSI artifacts
+        erb_file(File.join(VANAGON_ROOT, "resources/windows/wix/service.component.wxs.erb"), File.join(workdir, "service.#{name}.wxs"), false, { :binding => binding })
```

Comment:
@MikaelSmith we don't have access to the package_type at this moment in time. We could finagle it out of the binding that gets passed in, but we figured it wouldn't do any hard to generate both. The only thing that might be an issue would be missing variables... but I'm not sure if that would even throw an issue unless we explicitely said to